qiskit/wrapper/_monitor.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+
+"""A module for monitoring various qiskit functionality"""
+
+import sys
+import time
+import threading
+
+
+def job_monitor(job, interval=2, monitor_async=False):
+    """Monitor the status of a IBMQJob instance.
+
+    Args:
+        job (BaseJob): Job to monitor.
+        interval (int): Time interval between status queries.
+        monitor_async (bool): Monitor asyncronously.
+
+    Notes:
+        This function blocks output until the job has completed,
+        and is therefore useful in scripts, otherwise use the
+        Jupyter notebook magic %%qiskit_job_status.
+    """
+
+    def _checker(job, interval):
+        status = job.status()
+        msg = status.value
+        prev_msg = msg
+        msg_len = len(msg)
+
+        sys.stdout.write('\r%s: %s' % ('Job Status', msg))
+        while status.name not in ['DONE', 'CANCELLED', 'ERROR']:
+            time.sleep(interval)
```

Comment:
rather than use string can we import JobStatus here

Reply:
I am printing to the screen, so kind need a string.